src/components/ListBox/ListBox.js

```diff
@@ -82,12 +84,18 @@ ListBox.propTypes = {
    * `inline` as an option.
    */
   type: ListBoxType.isRequired,
+
+  /**
+   * Specify the "aria-label" of the ListBox.
+   */
+  ariaLabel: PropTypes.string,
 };
 
 ListBox.defaultProps = {
   innerRef: () => {},
   disabled: false,
   type: 'default',
+  ariaLabel: 'ListBox',
```

Comment:
Should we have a default value for this, or just leave it blank?

Reply:
Having a default value here doesn't hurt anything and it will help users that are using screen readers.

Reply:
It's also worth noting that many other Carbon components have a default value for the aria-label.

Reply:
@zgrossbart totally understand, it might help if the label itself is more descriptive then if the intent is to aid the user here by default. List Box is something that could cause confusion as that term is not very common, or at least that's my assumption.

Reply:
I based the defaults on the other defaults which were very simple.  Just identifying what the element was.  I'm happy to change it.  Do you have another suggestion?

Reply:
How about "Choose an item" or "Choose an option"? Just basing off of: https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

Reply:
Sounds good to me.